### ISSUE TITLE
Selected items should be announced by screen readers

### ISSUE THREAD

**Issue by jivanova** [telerik/kendo-ui-core#4020]
### Bug report

[DEMO](https://demos.telerik.com/kendo-ui/multiselect/index)

### Reproduction of the problem
TAB to focus the multiselect widget

### Current behavior
"Listbox collapse" is announced by the screen readers

### Expected/desired behavior

[DOJO](https://dojo.telerik.com/opUKi)

Value is announced by the screen reader. Suggested workaround in the dojo

### Environment

* **Kendo UI version:** 2018.1.221
* **Browser:** all
